bears/python/PyCheckBear.py

```diff
@@ -0,0 +1,40 @@
+import shlex
+
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.DistributionRequirement import (
+        DistributionRequirement)
+
+
+@linter(executable='pychecker',
+        output_format='regex',
+        output_regex=r'(?P<filename>\w+\.py):(?P<line>\d+): '
+                     r'(?P<message>.*)')
+class PyCheckerBear:
```

Comment:
Aren't the names inconsistent? Here you have named it `PyCheckerBear` but you have named the file as `PyCheckBear.py`

Reply:
thats by mistake i think let me correct btw thanks for pointing out